docs/GoogleSubscriber-streamingsource.md

```diff
@@ -28,6 +28,25 @@ It can be found on the Dashboard in the Google Cloud Platform Console.
 
 **Subscription**: Name of the Google Cloud PubSub subscription to subscribe.
 If the subscription needs to be created then the topic to which the subscription will belong must be provided.
+Naming Convention for Subscription:   
+                        
+    Not begin with the string goog.
```

Comment:
Are the pub/sub topic naming requirements documented somewhere? If yes, let's add a link to the documentation.

Reply:
Yes, cloud documentation link added to readme